yay blogposts. boo messiness.

bluesoradev opened the pull request:
idk about ascendproject/_posts lol

Commit:
wtf am i committing to?

Commit:
O_o

Commit:
newcommit

Commit:
yup

Commit:
humMerge branch 'gh-pages' of https://github.com/mozilla/ascendproject into gh-pages

Commit:
mkay

Commit:
Merge branch 'gh-pages' of https://github.com/mozilla/ascendproject into gh-pages
trying to retreive _posts/app-closed-time-4-round-2 :/

lsblakk commented:
Hey Sofie,  you seem to be erasing a bunch of blog posts from the top level _posts directory which should not be removed.  Can you restore those and remove them from this PR?  Also check out your build results, as they did not pass tests.

Commit:
again
Merge branch 'gh-pages' of https://github.com/mozilla/ascendproject into gh-pages

Commit:
o_O
